cpp/modmesh/universe/pymod/wrap_World.cpp

```diff
@@ -104,6 +104,106 @@ WrapVector3d<T>::WrapVector3d(pybind11::module & mod, const char * pyname, const
     // clang-format on
 }
 
+template <typename T>
+class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapPointPad
+    : public WrapBase<WrapPointPad<T>, PointPad<T>>
+{
+
+public:
+
+    using base_type = WrapBase<WrapPointPad<T>, PointPad<T>>;
+    using wrapped_type = typename base_type::wrapped_type;
+
+    using value_type = typename base_type::wrapped_type::value_type;
+
+    friend typename base_type::root_base_type;
+
+protected:
+
+    WrapPointPad(pybind11::module & mod, char const * pyname, char const * pydoc);
+};
+/* end class WrapPointPad */
+
+template <typename T>
+WrapPointPad<T>::WrapPointPad(pybind11::module & mod, const char * pyname, const char * pydoc)
+    : base_type(mod, pyname, pydoc)
+{
+    namespace py = pybind11;
+
+    // Constructors
+    (*this)
+        .def(py::init<uint8_t>(), py::arg("ndim"))
+        .def(py::init<SimpleArray<T> &, SimpleArray<T> &, bool>(),
+             py::arg("x"),
+             py::arg("y"),
+             py::arg("clone"))
```

Comment:
For now, make `clone` an explicit argument without a default.

Reply:
I am curious what `clone` is used for and why we need to make it explicit.

Reply:
If `clone` is true, share the inner memory buffer.  Otherwise, copy the data.

I haven't made my mind which way is more useful.  So before I have a feeling for it, it is left explicit without a default value.